src/AddTodoForm.jsx

```diff
@@ -1,15 +1,26 @@
+import React, { useState } from "react";
+
+
+
+function AddTodoForm ({onAddTodo}) {
+
+    const [todoTitle, setTodoTitle] = React.useState('')
+
+    function handleTitleChange (event) {
+        const newTodoTitle = event.target.value;
+        console.log(newTodoTitle)
+        setTodoTitle(newTodoTitle)
+    }
 
-function AddTodoForm (props) {
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const todoTitle = event.target.elements.title.value;
-        console.log(todoTitle)
-        props.onAddTodo(todoTitle)
+        onAddTodo({title: todoTitle, id: Date.now()});
+        setTodoTitle(' ');
```

Comment:
You are setting the title to a string with space

```suggestion
        setTodoTitle('');
```